.changeset/mean-olives-watch.md

```diff
@@ -0,0 +1,5 @@
+---
+'@sumup/circuit-ui': major
+---
+
+Removed the `RadioButton`'s deprecated `children` prop. Use the `label` prop (now required) instead.
```

Comment:
Added in the migration guide 😉 I can also mention it in the changelog, sure

Reply:
Ah, I missed that. Excellent work, carry on.

Reply:
Added a small mention of the RadioButtonGroup in [7d9f082](https://github.com/sumup-oss/circuit-ui/pull/1512/commits/7d9f08207bd803da4acf5cce332badfcce5e4a87) for clarity (in case someone goes through the changelog instead of the migration guide)